routers/payments.py

```diff
@@ -0,0 +1,66 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/payments")
+async def get_payments():
+    """
+    List payments
+
+        Args (optional):
+            fk_booking_id (int): id of booking that payment correspond to
+            sum (float): sum of payment
```

Comment:
> And `Args (optional)` means that all parameters are optional?

Yes, I thought it would be easier for users instead of writing in each variable

> is it possible use case where you want to retrieve payments based on sum of payment (I could imagine price range).

Agree. I'll add options.